Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -62,6 +62,9 @@ static FluidRendering()
             context.AmbientValues[ViewPath] = path;
             context.AmbientValues.Add("Sections", new Dictionary<string, IList<Statement>>());
 
+            var viewsPath = Path.Combine(_hostingEnvironment.ContentRootPath, "Views");
```

Comment:
Here just reuse the existing `fileProvider` variable. You can then optionally wrap it in another one that will convert the calls to something like `/Shared`. Maybe there is already a FileProvider implementation that does it.